docs/Windows.md

```diff
@@ -78,7 +78,7 @@ set swift_source_dir=path-to-directory-containing-all-cloned-repositories
 ```
 
 ### 5. Build CMark
-- This must be done from within a developer command prompt, and could take up to 10 minutes.
+- This must be done from within a developer command prompt. CMark is a fairly small project, and it should only take a few minutes to build.
```

Comment:
Since we're nitpicking: I was actually curious and looked this up. Apparently, my change as written is correct; you need a comma before a conjunction when it's connecting two independent clauses. As such, the old version,
> prompt, and could
was incorrect, since that's a dependent clause, but my modification is actually correct.

However, the question is more whether it should be two independent clauses. I think it's probably clearer/more natural as a dependent clause, so I've changed it to match (and now the comma is gone.) I also went through and addressed the same concern in other places in the same file.